provisioning/miniprov/hare_mp/main.py

```diff
@@ -523,16 +523,31 @@ def generate_config(url: str, path_to_cdf: str) -> None:
     if path:
         path += os.pathsep + '/opt/seagate/cortx/hare/bin/'
     python_path = os.pathsep.join(sys.path)
+
+    conf = ConfStoreProvider(url)
+    hostname = conf.get_hostname()
+    save(f'{conf_dir}/node-name', hostname)
+
     cmd = ['configure', '-c', conf_dir, path_to_cdf]
     execute(cmd, env={'PYTHONPATH': python_path, 'PATH': path,
                       'LC_ALL': "en_US.utf-8", 'LANG': "en_US.utf-8"})
     utils.import_kv(conf_dir)
     cmd = ['update-consul-conf']
     execute(cmd, env={'PYTHONPATH': python_path, 'PATH': path,
                       'LC_ALL': "en_US.utf-8", 'LANG': "en_US.utf-8"})
-    conf = ConfStoreProvider(url)
-    hostname = conf.get_hostname()
-    save(f'{conf_dir}/node-name', hostname)
+
+    # Create motr confd.xc
+    with open(f'{conf_dir}/confd.dhall', 'r') as confdhall:
```

Comment:
@mssawant Do you really need this thing?

I believe this is already done by `configure` script.:
https://github.com/Seagate/cortx-hare/blob/220d3202f74bb32be6479956d3cbce4b3c28a52e/hax/helper/configure.py#L70-L81

Reply:
Indeed! somehow when I was testing I didn't notice confd.xc getting created. probably  I made some mistake in the rpms that I installed.
We still have 1 problem, configure.py does not import consul kv due to which `update-consul-conf is not able to add services to `consul-server-conf.json`.
Let's abandon this PR. probably we will have to add consul kv import to configure.py but that will add dependency on consul to be running, which is true in the current situation but I am wondering if we should add this dependency on configure.py.